internal/core/src/common/Schema.cpp

```diff
@@ -69,25 +68,26 @@ Schema::ParseFrom(const milvus::proto::schema::CollectionSchema& schema_proto) {
                 auto metric_type = GetMetricType(index_map.at("metric_type"));
                 schema->AddField(name, field_id, data_type, dim, metric_type);
             }
+        } else if (datatype_is_string(data_type)) {
+            auto type_map = RepeatedKeyValToMap(child.type_params());
+            AssertInfo(type_map.count("max_length_per_row"), "max_length_per_row not found");
+            auto max_len = boost::lexical_cast<int64_t>(type_map.at("max_length_per_row"));
```

Comment:
we'd better define a const char* instead of using "max_length_per_row" every where

Reply:
fix